rhelai-validation/tasks/deploy_vm.yaml

```diff
@@ -0,0 +1,25 @@
+---
+- name: Git checkout the rhoso-rhelai roles
+  ansible.builtin.git:
+    repo: "{{ deploy_rhelai_repo }}"
+    dest: "{{ lookup('env', 'HOME') }}/rhoso-rhelai"
+    version: "{{ deploy_rhelai_branch }}"
+- name: Set rhoso-rhelai role paths
+  ansible.builtin.set_fact:
+    deploy_rhelai_gpu_roledir: "{{ lookup('env', 'HOME') }}/rhoso-rhelai/nested-passthrough/ansible/setup_gpu"
+    deploy_rhelai_setup_roledir: "{{ lookup('env', 'HOME') }}/rhoso-rhelai/nested-passthrough/ansible/setup_rhelai"
+
+- name: Collect GPU facts required for RHELAI deployment
+  ansible.builtin.include_role:
+    name: "{{ deploy_rhelai_gpu_roledir }}"
+    tasks_from: gpuinfo.yml
+- name: Copy oclogin script for use by deploy_rhelai tasks
+  ansible.builtin.copy:
+    src: "{{ deploy_rhelai_setup_roledir }}/files/oclogin"
+    dest: ~/oclogin
+    mode: "0600"
+
+- name: Hand off to deploy_rhelai from setup_rhelai Role
+  ansible.builtin.include_role:
+    name: "{{ deploy_rhelai_setup_roledir }}"
+    tasks_from: deploy_rhelai.yml
```

Comment:
This ultimately uses `oc rsh openstack ....` commands, which I think could just be `openstack` in the test-container. I'll be looking at this ASAP to see what adjustments are required.

https://github.com/openstack-k8s-operators/rhoso-rhelai/blob/main/nested-passthrough/ansible/setup_rhelai/tasks/deploy_rhelai.yml#L62

Reply:
Yes. The container only needs to have the same facilities as the openstack-client container we are otherwise using.

Reply:
We're mounting into the test pod `clouds.yaml` under `/etc/openstack/clouds.yaml` [1]. 

[1] https://github.com/openstack-k8s-operators/test-operator/blob/d71f525e3a4587610bdb73e185d768f464b8f439/pkg/ansibletest/volumes.go#L162